Tasks/MSBuild/Helpers.ps1

```diff
@@ -60,6 +60,11 @@ function Invoke-BuildTools {
         [switch]$NoTimelineLogger
     )
 
+    if ($NuGetRestore)
+    {
+        Write-Warning (Get-LocalizedString -Key "The `"Restore NuGet Packages`" option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.")
```

Comment:
Updating the template should be part of this overall effort.

Sent from Outlook Mobile<https://aka.ms/blhgte>



On Tue, Jan 26, 2016 at 3:37 PM -0800, "Matt Cooper" <notifications@github.com<mailto:notifications@github.com>> wrote:


In Tasks/MSBuild/Helpers.ps1<https://github.com/Microsoft/vso-agent-tasks/pull/1135#discussion_r50919945>:

> @@ -60,6 +60,11 @@ function Invoke-BuildTools {
>          [switch]$NoTimelineLogger
>      )
>
> +    if ($NuGetRestore)
> +    {
> +        Write-Warning (Get-LocalizedString -Key "The `"Restore NuGet Packages`" option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.")


@ericsciple<https://github.com/ericsciple> it would be awesome to simply not have the checkbox in the new task, and set up the default Visual Studio Build template to include a NuGet Installer step. Is that possible?

-
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/vso-agent-tasks/pull/1135/files#r50919945>.